Populate variables for header meta tags

Jwager-SBRx opened the pull request:
Populate pageTitle and pageDesc fields for inclusion in header meta tags

Jwager-SBRx commented:
cf2a0eb overrides/supplements changes in 1c52bbb

jonesde commented:
There are a number of changes here for whitespace only, please remove those changes.

In the content.xml screen it looks like you are using the first (or zero-th rather) entry from the sri.screenUrlInfo.extraPathNameList list for the pageTitle and pageDesc. Wouldn't it be better to use the last entry in that list, which is the current wiki page's name (possibly preceded by a path within the wiki space)? Also, it looks like you are setting pageTitle and pageDesc to the same value but in different ways... might as well do something like set the pageTitle and then set pageDesc from pageTitle.

In both files there is an if/then/else block and both are formatted oddly. To make it more like C or Java code I usually format them this way:

```
<if><then>
    ...
</then><else>
    ...
</else></if>

```

Jwager-SBRx commented:
The white space and If...then...else formatting were Intellij's choice. I
will correct those. I assume that there's a way to set up default styles.
Do you have a set of standards you would like me to implement?

I had considered the array referencing you suggest, but since I barely know
Java and don't know Groovy, and am not certain which is involved on the
pages in question, I went with what worked. I will look into that further.

On the context screen, I am setting both the title and the description to
the same value. That is intentional for those screens.
On the product screen, the product.Productname and the product.description
contain different values (When the description contains anything at all.) I
don't see an issue there. Am I missing something?

I will make the changes on the first two items and do another pull request.

Thanks,
Joe

On Fri, Sep 3, 2021 at 3:44 PM David E. Jones ***@***.***>
wrote:

> There are a number of changes here for whitespace only, please remove
> those changes.
>
> In the content.xml screen it looks like you are using the first (or
> zero-th rather) entry from the sri.screenUrlInfo.extraPathNameList list for
> the pageTitle and pageDesc. Wouldn't it be better to use the last entry in
> that list, which is the current wiki page's name (possibly preceded by a
> path within the wiki space)? Also, it looks like you are setting pageTitle
> and pageDesc to the same value but in different ways... might as well do
> something like set the pageTitle and then set pageDesc from pageTitle.
>
> In both files there is an if/then/else block and both are formatted oddly.
> To make it more like C or Java code I usually format them this way:
>
> <if><then>
>     ...
> </then><else>
>     ...
> </else></if>
>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/moqui/PopRestStore/pull/63#issuecomment-912826648>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARRT3IEHOE2SN3HXQR4KTULUAE6SHANCNFSM5DMMY65Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

Jwager-SBRx commented:
I have changed the white space reformatting - though it looks like IntelliJ is still fighting me on some.
I have reformatted the <if><then>...<else> tags.
I have changed the Array lookup selection.
I tried to create a new pull request, but cannot tell if it was created or not.
